.circleci/config.yml

```diff
@@ -1426,6 +1426,18 @@ workflows:
       - fuzz-op-node
       - fuzz-op-chain-ops
       - bedrock-markdown
+      - go-lint:
```

Comment:
The upstream tests for the bindings generate the bindings, and make sure that there is no diff based on what's in the source tree -- I think this is a good idea.  Can we add there here too?

Reply:
This requires us to build a specific image for the testing purpose, since we use the abigen from erigon.

Reply:
The test is added. https://app.circleci.com/pipelines/github/bobanetwork/v3-anchorage/164/workflows/82ad7198-25d8-4b1a-9f5c-43a596a1070f/jobs/9710